pyproject.toml

```
@@ -34,7 +34,8 @@ dependencies = [
     "Flask-Cors>=3.0.8,<4",
     "Flask-Limiter>=2.6,<3",
     "Flask-Mail>=0.9.1,<1",
-    "Flask-Migrate>=2.5.3,<5",  # Not following semantic versioning (e.g. https://github.com/miguelgrinberg/flask-migrate/commit/1af28ba273de6c88544623b8dc02dd539340294b)
+    "Flask-Migrate>=2.5.3,<5",
+    # Not following semantic versioning (e.g. https://github.com/miguelgrinberg/flask-migrate/commit/1af28ba273de6c88544623b8dc02dd539340294b)
```

This comment should go above, I guess autoformat was involved.
```suggestion
    # Not following semantic versioning (e.g. https://github.com/miguelgrinberg/flask-migrate/commit/1af28ba273de6c88544623b8dc02dd539340294b)
    "Flask-Migrate>=2.5.3,<5",
```